cscore/src/main/native/include/cscore_raw.h

```diff
@@ -9,8 +9,19 @@
 
 #ifdef __cplusplus
 #include "cscore_oo.h"
-#endif
-
+/**
```

Comment:
I don't understand the duplication here.  Why is this necessary?

Reply:
typedefed structs are normal in C code, but when this header is compiled as C++, clang-tidy requires `using` instead of `typedef` as a general rule.

Reply:
This is uncommon enough in our code base that I think we could just NOLINT it.

Reply:
One alternative to avoid the NOLINT and duplication could be to separate the typedef and struct definition.